traffic_ops/app/lib/Schema/Result/Deliveryservice.pm

```diff
@@ -390,8 +401,12 @@ __PACKAGE__->add_columns(
   { data_type => "text", is_nullable => 1 },
   "logs_enabled",
   { data_type => "boolean", default_value => \"false", is_nullable => 1 },
+  "multi_site_origin_algorithm",
```

Comment:
It is auto generated. Wouldn't it be better to except it or pull pr 379 (https://github.com/apache/incubator-trafficcontrol/pull/379) and merge?